bin/dns/zone/import/index.js

```diff
@@ -33,70 +33,92 @@ const supported_label = supported_types
     .map(x => x.toUpperCase())
     .join(', ');
 
+const equalRrset = (a, b) => a.record.length == b.record.length &&
+    a.record.every(x => b.record.some(y => y.content == x.content)) &&
+    b.record.every(x => a.record.some(y => y.content == x.content));
+
+const zonefile2rrset = (fname, dnsName) => {
```

Comment:
The extracted logic of converting the zonefile structure from the library to our own structure. I have developed it and would like to keep it, although the initial cause of the change is no longer present (approach changed).